src/python/pants/backend/python/rules/coverage.py

```diff
@@ -294,8 +294,20 @@ async def generate_coverage_reports(
     )
 
     processes = []
-    report_types = coverage_subsystem.reports
-    for report_type in report_types:
+    report_types = []
+    coverage_reports: List[CoverageReport] = []
+    for report_type in coverage_subsystem.reports:
+        if report_type == CoverageReportType.RAW:
+            coverage_reports.append(
+                FilesystemCoverageReport(
+                    report_type=CoverageReportType.RAW,
+                    result_digest=input_digest,
```

Comment:
The `input_digest` has far more than you want. Instead, use `merged_coverage_data.coverage_data`. 

(Speaking of which, that's not a great field name. Feel free to rename to `merged_coverage_data.digest`.)

Reply:
yes... testing locally I noticed that `.coveragerc` & `coverage.pex`  show up in addition to `.coverage`